### ISSUE TITLE
feature request: add command line parameter for sso provider

### ISSUE THREAD

**Issue by frisbee23** [oracle/oci-cli#618]
hi; 
when signing in via the webinterface, and using sso as part of authorization, it's possible to specify in the URL the sso provider and tenant.
https://cloud.oracle.com/?tenant=mycompany&provider=some-sso-provider

the oci-cli has a parameter for tenant '--tenant' but no parameter for provider:

# a little slightly faked example:
$ oci session authenticate --tenant mycorp --provider some-sso-provider --region...
Usage: oci session authenticate [OPTIONS]

Error: no such option: --provider

could someone please add that? i guess it would save countless workers a little time each day.
or maybe it's

**Comment by kk-hiraskar** [comment]
Yes this is a must needed feature, & already it's been more than 2 years since this request.

Maintainers, could you please comment, any reason this feature implementation is not yet picked??

**Comment by atnoya** [comment]
I agree this would be very useful or at least a way to set it via configuration

**Comment by atnoya** [comment]
Actually, this is solved in 3.71.1, the parameter name is `--identity-provider-name`

**Comment by PechimuthuMithil** [comment]
Thanks @atnoya !
@frisbee23 @kk-hiraskar This is resolved with the addition of `--identity-provider-name` **optional** parameter from version 3.71.1. 
Closing this issue.
